g.mt

```diff
@@ -0,0 +1,38 @@
+abstract Interactable {
```

Comment:
maybe move this to a folder like `lang-tests` similar to `mir-tests`

Reply:
it's a temporary file, i literally just uploaded whatever i had locally